src/parameters.jl

```diff
@@ -319,6 +315,16 @@ function MOI.set(
     return
 end
 
+function MOI.set(
+    model::Optimizer,
+    ::ForwardConstraintSet,
+    ci::MOI.ConstraintIndex{MOI.VariableIndex,MOI.Parameter{T}},
+    set::MOI.Parameter,
+) where {T}
+    model.input_cache.dp[ci] = set.value
+    return
+end
+
```

Comment:
doesnt it exist already?
https://github.com/jump-dev/DiffOpt.jl/blob/95927fae094da6603cfe02f608d7bf92898bb70d/src/parameters.jl#L307

Reply:
Oh, fine!